src/test/components/Component/module.cpp

```diff
@@ -0,0 +1,39 @@
+#include <pal.h>
+#include "Class.h"
+
+using namespace std::string_view_literals;
+
+extern "C"
+{
+    xlang_result XLANG_CALL lib_get_activation_factory(xlang_string class_name, xlang_guid const& iid, void **factory) noexcept
+    try
+    {
+        *factory = nullptr;
+
+        char16_t const* buffer{};
+        uint32_t length{};
+        winrt::check_hresult(xlang_get_string_raw_buffer_utf16(class_name, &buffer, &length));
+        std::u16string_view name{ buffer, length };
+        
+        if (name == u"Component.Class"sv)
+        {
+            // TODO: Unify xlang guid and winrt guid

```

Comment:
have we given any thought to this?  usings in pal.h?

Reply:
Since this code would eventually be generated automatically by the language projection, much as C++/WinRT generates WINRT_GetActivationFactory implementation today, does it matter?